src/main/scala/cromwell/Boot.scala

```diff
@@ -0,0 +1,65 @@
+package cromwell
+
+import java.io.File
+import java.nio.file.{Files, Paths}
+
+import akka.actor.{Props, ActorSystem}
+import akka.io.IO
+import akka.pattern.ask
+import akka.util.Timeout
+import com.typesafe.config.{Config, ConfigFactory}
+import com.wordnik.swagger.model.ApiInfo
+import cromwell.engine.WorkflowManagerActor
+import cromwell.webservice._
+import spray.can.Http
+
+import scala.concurrent.duration._
+import scala.reflect.runtime.universe._
+import scala.util.{Failure, Success}
+
+
+object Boot extends App {
+
+  private def startup(): Unit = {
+    val conf = ConfigFactory.parseFile(new File("/etc/cromwell.conf"))
+
+    // we need an ActorSystem to host our application in
+    implicit val system = ActorSystem("cromwell")
+
+    val swaggerConfig = conf.getConfig("swagger")
+    val swaggerService = new SwaggerService(
+      swaggerConfig.getString("apiVersion"),
+      swaggerConfig.getString("baseUrl"),
+      swaggerConfig.getString("apiDocs"),
+      swaggerConfig.getString("swaggerVersion"),
+      Seq(typeOf[CromwellApiService]),
```

Comment:
I don't know enough about Swagger to tell you if what I'm about to say is a good idea or not but a habit I'm trying to push is that when you're instantiating a Seq to use Vector unless there's a reason not to.

Think of Seq as java List, and then List/Vector as e.g. ArrayList/LinkedList/etc. Seq's default implementation is List and historically most people just default to using List everywhere (or Seq) much like Java folks default to using ArrayList. But these days Vector is slightly more performant in almost all cases.